Consolidate Complaint CWS FAQs and CWS Review Process (#7701)
GoogleChrome/developer.chrome.com: site/en/docs/webstore/review-process/index.md

Support tickets must be opened using the [One Stop Support][header-support] form, but once a
ticket is opened all communication takes place over email.

{% Aside %}

> While the team does monitor and respond to various online forums where Chrome extensions are discussed, these platforms are primarily geared towards discussions about extension development and are not intended to be support channels. If you need to raise any concerns, it is recommended to use the CWS Support form.

Reviewer: ```suggestion
Though the team monitors and responds to various online forums where Chrome extensions are discussed, these platforms are primarily geared towards discussions about extension development and are not intended to be support channels. If you need to raise any concerns, it is recommended to use the CWS Support form.
```

difficult for the agents assisting you to find and keep track of all of the relevant information
about your issue.

{% Aside %}

> As a developer in the EU, if you have concerns related to the Platform To Business (P2B) regulations in Europe, you can raise them by filing through the [One Stop Support contact form][page-oss-form].

Reviewer: ```suggestion
As a developer in the European Union, if you have concerns related to the Platform To Business (P2B) regulations in Europe, you can raise them by filing through the [One Stop Support contact form][page-oss-form].
```
